data/source/MongoDb.php

```diff
@@ -212,53 +213,69 @@ public static function enabled($feature = null) {
 	 * @link http://php.net/mongo.construct.php PHP Manual: Mongo::__construct()
 	 * @param array $config Configuration options required to connect to the database, including:
 	 *        - `'host'` _string|array_: A string in the form of `'<host>'`, `'<host>:<port>'` or
-	 *          `':<port>'` indicating the host and/or port to connect to. When one or both are
-	 *          not provided uses general server defaults (if possible retrieved from the client
-	 *          implementation).
-	 *          Use the array format for multiple hosts:
-	 *          `array('167.221.1.5:11222', '167.221.1.6')`
+	 *          `':<port>'` indicating the host and/or port to connect to.
+	 *          Use the array format for multiple hosts: `['167.221.1.5:11222', '167.221.1.6']`
+	 *        - `'login'` _string_: Username to use when connecting to server. Defaults to `'root'`.
+	 *        - `'password'` _string_: Password to use when connecting to server. Defaults to `''`.
 	 *        - `'database'` _string_: The name of the database to connect to. Defaults to `null`.
 	 *        - `'timeout'` _integer_: The number of milliseconds a connection attempt will wait
 	 *          before timing out and throwing an exception. Defaults to `100`.
 	 *        - `'schema'` _\Closure_: A closure or anonymous function which returns the schema
 	 *          information for a model class. See the `$_schema` property for more information.
-	 *        - `'gridPrefix'` _string_: The default prefix for MongoDB's `chunks` and `files`
-	 *          collections. Defaults to `'fs'`.
-	 *        - `'replicaSet'` _string_: See the documentation for `Mongo::__construct()`. Defaults
-	 *          to `false`.
-	 *        - `'readPreference'` _mixed_: May either be a single value such as Mongo::RP_NEAREST,
```

Comment:
Including docs for every driver and uri option is too much work. We should point users to the general mongo documentation instead.

Reply:
All online documentation has been removed and references to the official php adapter and the original MongoDB documentation are in place.